lib/resetState.spec.ts

```diff
@@ -0,0 +1,84 @@
+import { resetState } from "./resetState";
+
+it("sets state id to identify state", () => {
+  const state = {
+    a: 1,
+    b: 2,
+    c: 3,
+  };
+
+  resetState(state);
+
+  expect((state as any)["___testStateStateId"]).toBeDefined();
+});
+
+it("sets state id as non enumerable", () => {
```

Comment:
```suggestion
it("adds STATE_ID_PROPERTY to original state as non enumerable", () => {
```
^ something like this? 🤔

Reply:
Updated